guides/preprocessing_layers.py

```diff
@@ -32,13 +32,15 @@
 These layers are for structured data encoding and feature engineering.
 
 - `CategoryEncoding` layer: turns integer categorical features into one-hot, multi-hot,
-or TF-IDF dense representations.
+or count dense representations.
 - `Hashing` layer: performs categorical feature hashing, also known as the "hashing
 trick".
 - `Discretization` layer: turns continuous numerical features into integer categorical
 features.
-- `StringLookup` layer: turns string categorical values into integers indices.
-- `IntegerLookup` layer: turns integer categorical values into integers indices.
+- `StringLookup` layer: turns string categorical values an encoded representation that can be
+read by an `Embedding` layer or `Dense` layer.
+- `IntegerLookup` layer: turns integer categorical values an encoded representation that can be
```

Comment:
Suggestion: "turns ... values _into_ an encoded representation"

```suggestion
- `IntegerLookup` layer: turns integer categorical values into an encoded representation that can be
```